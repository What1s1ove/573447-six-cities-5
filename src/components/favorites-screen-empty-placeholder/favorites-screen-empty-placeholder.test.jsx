import * as React from 'react';
import {render} from 'enzyme';
import FavoritesScreenEmptyPlaceholder from './favorites-screen-empty-placeholder';

describe(`FavoritesScreenEmptyPlaceholder component`, () => {
  it(`should render FavoritesScreenEmptyPlaceholder component`, () => {
    const component = render(<FavoritesScreenEmptyPlaceholder />);

    expect(component).toMatchSnapshot();
  });
});
