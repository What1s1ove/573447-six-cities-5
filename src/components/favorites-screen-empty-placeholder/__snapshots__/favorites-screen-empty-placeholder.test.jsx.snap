// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesScreenEmptyPlaceholder component should render FavoritesScreenEmptyPlaceholder component 1`] = `
Array [
  <h1
    class="visually-hidden"
  >
    Favorites (empty)
  </h1>,
  <div
    class="favorites__status-wrapper"
  >
    <b
      class="favorites__status"
    >
      Nothing yet saved.
    </b>
    <p
      class="favorites__status-description"
    >
      Save properties to narrow down search or plan yor future trips.
    </p>
  </div>,
]
`;
